### PR TITLE
Update auth-middleware.mdx

### PR DESCRIPTION
**Pull request by shawnesquivel**
typo on behavior
add comma to make code syntactically correct

### CODE REVIEW

File: docs/references/nextjs/auth-middleware.mdx [hunk 1]
```diff
@@ -30,7 +30,7 @@ With the recommended matcher, all routes are protected by Clerk's authentication
 
 ### Default behaviour
 
-If there is no `afterAuth` present in the middleware, then `authMiddleware` will fall back to its default behaviour. The default bahaviour is:
+If there is no `afterAuth` present in the middleware, then `authMiddleware` will fall back to its default behaviour. The default behaviour is:
```
Comment (alexisintech) [inline docs/references/nextjs/auth-middleware.mdx:33]:
```suggestion
If there is no `afterAuth` present in the middleware, then `authMiddleware` will fall back to its default behavior. The default behavior is:
```

**Review by alexisintech — CHANGES_REQUESTED** [review]

File: docs/references/nextjs/auth-middleware.mdx [hunk 2]
```diff
@@ -30,7 +30,7 @@ With the recommended matcher, all routes are protected by Clerk's authentication
 
 ### Default behaviour
 
-If there is no `afterAuth` present in the middleware, then `authMiddleware` will fall back to its default behaviour. The default bahaviour is:
+If there is no `afterAuth` present in the middleware, then `authMiddleware` will fall back to its default behavior. The default behavior is:
```
Comment (S3Prototype) [inline docs/references/nextjs/auth-middleware.mdx:33]:
```suggestion
If there is no `afterAuth` present in the middleware, then `authMiddleware` will fall back to its default behavior:
```

**Review by S3Prototype — APPROVED** [review]
Approved with one suggested change 👍

**Review by shawnesquivel — COMMENTED** [review]
added requested changes

**Review by alexisintech — APPROVED** [review]
thanks so much for the contribution 😸💖

**Comment by shawnesquivel** [comment]
> thanks so much for the contribution 😸💖

My pleasure!

**Pull request merged**